static/js/settings_org.js

```diff
@@ -139,6 +150,51 @@ function set_add_emoji_permission_dropdown() {
     $("#id_realm_add_emoji_by_admins_only").val(get_property_value("realm_add_emoji_by_admins_only"));
 }
 
+exports.setup_msg_edit_limit_dropdown_values = function () {
+    exports.msg_edit_limit_dropdown_values = {
+        any_time: {
+            text: i18n.t("Any time"),
+            seconds: 0,
+        },
+        never: {
+            text: i18n.t("Never"),
+        },
+        upto_two_min: {
+            text: i18n.t("Up to 2 minutes after posting"),
+            seconds: 2*60,
+        },
+        upto_ten_min: {
+            text: i18n.t("Up to 10 minutes after posting"),
+            seconds: 10*60,
+        },
+        upto_one_hour: {
+            text: i18n.t("Up to 1 hour after posting"),
+            seconds: 60*60,
+        },
+        upto_one_day: {
+            text: i18n.t("Up to 1 day after posting"),
+            seconds: 24*60*60,
+        },
+        upto_one_week: {
+            text: i18n.t("Up to 1 week after posting"),
+            seconds: 7*24*60*60,
```

Comment:
It kinda sucks that this creates so many translatable strings.  I think we might want to split it up, e.g.

i18n.t("Up to __time_period__ after posting", {time_period: period}), and then have a separate block for the time periods (which would at least mean the translated strings are reusable if we add other settings with a similar structure).

@gnprice @rishig thoughts?

Reply:
@timabbott  Still, we have to pass strings as `time_period` like `time_period: "2 minutes"`, `time_period: "1 day"`, etc. and then we can't mark "minutes", "hour", "week", "day" for translation. Or am I not understanding your suggestion?

Reply:
Tim: sounds good. 
Shubham: that's correct. (It's hard to split the numbers from the day/week/etc since pluralization works differently in different languages. E.g. some languages have a different plural ending for 2 of something than 3 of something, and some don't.)